code-build.js

```diff
@@ -21,20 +21,23 @@ function runBuild() {
   const sdk = buildSdk();
 
   // Get input options for startBuild
-  const params = inputs2Parameters(githubInputs());
+  const [hideLog, p] = githubInputs();
```

Comment:
There is a recent change that also introduced parameters to be used on a GitHub Action side. 

https://github.com/aws-actions/aws-codebuild-run-build/blob/main/code-build.js#L25-L28

Could you follow this convention to consolidate the `hideLog` to the `config` obect?